Data Management Blog Post

alexkim205 opened the pull request:
## Changes

Putting out a first iteration of the launch week "Data Management" blog post. Please feel free to give generous amounts of feedback!

Closes https://github.com/PostHog/posthog.com/issues/3068

## Blocking changes
- [ ] https://posthog.com/docs/user-guides/events page needs to be updated with documentation for event and property definitions.

## Checklist
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)
- [x] Words are spelled using American English
- [x] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["andy-vandervell"]
```
```suggestion
author: ["alex-kim"]
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+categories: ["Product updates"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continue to grow into the billions.
```
```suggestion
As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+## What is an Event?
+
+[Events](https://posthog.com/docs/user-guides/events) are the basic building blocks of data that is sent to PostHog. Events can capture a wide variety of different interactions, from a new user signing up to capturing a server event when an API endpoint is called.
```
```suggestion
[Events](/docs/user-guides/events) are the basic building blocks of data that is sent to PostHog. Events can capture a wide variety of different interactions, from a new user signing up to capturing a server event when an API endpoint is called.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+[Events](https://posthog.com/docs/user-guides/events) are the basic building blocks of data that is sent to PostHog. Events can capture a wide variety of different interactions, from a new user signing up to capturing a server event when an API endpoint is called.
+
+[Event Properties](https://posthog.com/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
```
```suggestion
[Event Properties](/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+### Creating definitions with the API
+
+You can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](https://posthog.com/docs/api/event-definitions) and [`property_definitions`](https://posthog.com/docs/api/property-definitions) API endpoints.
```
```suggestion
You can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+We're excited to get these data management features in, but we're just scratching the surface. We've prepared some documentation to help you get started with creating definitions and potential use-cases.
+
+As always, we welcome any feedback you may have about this feature. Feel free to open an issue in our Github repo or give us a shout in our PostHog Users slack channel.
```
```suggestion
As always, we welcome any feedback you may have about this feature. Feel free to open an issue in [our Github repo](https://github.com/PostHog/posthog) or give us a shout in our [PostHog community Slack channel](/slack).
```

andyvan-ph reviewed (commented):
First pass with a few small tweaks

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-07
```
```suggestion
date: 2022-03-23
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
+
+This is the first step in a continuous effort to help make organizing your data easier:
+
+- Creating definitions for events and event properties
+- Managing data on the fly while building insights
+- New UI (Data Management tab and revamped definitions page)
+- Verifying trusted definitions
+- Categorizing your definitions with tags
+- Giving custom names and descriptions to events and properties
+- Making the right events and properties discoverable with event metrics
```
```suggestion
This is the first step in a continuous effort to help make organizing your data easier. With the new Data Management tab and UI you can:

- Create definitions for events and event properties
- Manage data on the fly while building insights
- Verify trusted definitions which are prioritzed
- Organize your definitions with tags
- Give custom names and descriptions to events and properties
- Make the right events and properties discoverable with event metrics
```

andyvan-ph replied:
Tweaked this section a little just to make flow a little better. I think it would be a good idea to have a screenshot of the Data Management tab below this list. Could be a short gif/video if that's practical?

alexkim205 replied:
Yes! great idea

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-07
+title: You can now organize your PostHog data
```
We probably need to work on the headline a bit as this implies you couldn't organize data before.

Suggestions welcome as nothing springs to mind atm!

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
```
Perhaps we could expand a little more on the whole 'data explosion' theme and its impacts, why we wanted to solve it?

paolodamico replied:
Avoid PH acronym?

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
```
Maybe some examples with numbers? e.g. certain customer who has ~30k unique events. Another approach could be to show a before/after picture?

alexkim205 replied:
Nice, I wasn't sure if we could show data from prod here but I guess just the counts are fine.

timgl replied:
@jamesefhawkins are you happy with these numbers being public

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
+
+This is the first step in a continuous effort to help make organizing your data easier:
```
I would consider targeting this based on use cases & problems rather than functionality (we can list functionality in the feature page). Example: engineers/PMs sharing context so that marketing users can figure out their attributions.

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+## How to use Data Management as a team (Best practices)
+
+1. **Rule of thumb**: The more definitions you define, the easier it will be for your teammates to understand what each event and property is describing. 
```
```suggestion
1. **Rule of thumb**: The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
```

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+1. **Rule of thumb**: The more definitions you define, the easier it will be for your teammates to understand what each event and property is describing. 
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PH instance sending sign up events as `sign up`. A new teammate using PH will find it impossible to differentiate between the two, leading to incorrect queries and onboarding woes. Defining two separate definitions (with differ ...
```
```suggestion
Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PH instance sending sign up events as `sign up`. Or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, potentially leading to incorrect conclusions and frustration. Describing these events property can ensure your teammates find exactly what they're looking for.
```

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+2. **Rule of thumb**: Official organization-wide events should be verified so that teammates can discover and use them easily. 
+
+Verified events will be prioritized in filters and other selection components to signal to the rest of the team that this event should be used in favor of other similar events.
```
Maybe add an example?

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+Verified events will be prioritized in filters and other selection components to signal to the rest of the team that this event should be used in favor of other similar events.
+
+3. **Rule of thumb**: Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page. 
```
I would take a very opinionated approach here where we share best practices on what to tag. Example: scope tagging (`product`, `marketing`, `website`, ...)

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+We're excited to get these data management features in, but we're just scratching the surface. We've prepared some documentation to help you get started with creating definitions and potential use-cases.
+
+As always, we welcome any feedback you may have about this feature. Feel free to open an issue in [our Github repo](https://github.com/PostHog/posthog) or give us a shout in our [PostHog community Slack channel](/slack).
```
```suggestion
As always, we welcome any feedback you may have about this feature. Feel free to open an issue in [our Github repo](https://github.com/PostHog/posthog) or give us a shout in our [PostHog community Slack channel](/slack). Or join us directly for a [call](https://calendly.com/posthog-feedback) with our Product & Engineering team.
```

paolodamico commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,83 @@
+
+## We've moved some things around
+
+- The "Events & Actions" tab has been renamed to "Live Events".
```
Suggestion. Highlight the new stuff and the rationale.
```suggestion
- Introducing "Live Events" which will help you debug your incoming event data in near real-time.
```

paolodamico reviewed (commented):
Some suggestions inline, hope it helps.

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+2. Data decays quickly, and most of the time fails to stay up-to-date with instrumentation.
+3. Event-level context in insights and dashboards is difficult to share with external people and new team members.
+
+Fret no more because we've been working hard to deliver a solution to your data management woes!
```
```suggestion
Well, fret no more because we've been working hard to deliver a solution to your data management woes!
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+![PostHog - data management tab](../images/blog/data-management-feature/data_management_tab.png)
+
+## What is an Event?
```
```suggestion
## How to create event and event property definitions
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+[Event Properties](/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
+
+## How to create event and event property definitions
```
```suggestion
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+1. **Rule of thumb**: The more definitions you define, the easier it will be for your teammates to understand what each event and property is describing. 
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PH instance sending sign up events as `sign up`. Or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, potential ...
```
```suggestion
Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PostHog instance sending sign up events as `sign up`. Or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, potentially leading to incorrect conclusions and frustration. Describing these events properly can ensure your teammates find exactly what they're looking for.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+## What's next for Data Management?
+
+We're excited to get these data management features in, but we're just scratching the surface. We've prepared some documentation to help you get started with creating definitions and potential use-cases.
```
We should expand a little more on the roadmap for data management here. Just a taster to give people a sense of our direction.

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,95 @@
+categories: ["Product updates", "Launch week"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the double-digit billions. In just the last year (since we started measuring total events), we've gone from ~8B to ~36.5B total events ingested in PostHog Cloud!
```
```suggestion
PostHog is growing fast. In just the last year (since we started measuring total events), we've gone from ~8B to ~36.5B total events ingested in PostHog Cloud alone!
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,95 @@
+
+![PostHog - total events](../images/blog/data-management-feature/total_events.png)
+
+Data explosion is real and can be a nightmare to scale in the context of a product analytics tool. Furthermore, poor data management can seriously harm your business as it can lead to inaccurate analyses and poor business decisions.
```
```suggestion
This is great for us as a business, but data explosion is real and can be a nightmare to scale in the context of a product analytics tool. Furthermore, poor data management can seriously harm your business as it can lead to inaccurate analyses and poor business decisions.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,95 @@
+
+Well, fret no more because we've been working hard to deliver a solution to your data management woes!
+
+Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PostHog. 
```
```suggestion
Today, we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PostHog. 
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,95 @@
+
+Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PostHog. 
+
+This is the first step in a continuous effort to help make organizing your data easier. You can now:
```
```suggestion
This is the first release in our ongoing commitment to making organizing your data easier. In this first release, we're introducing a dedicated Data Management, allowing you to:
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+- Make the right events and properties discoverable with event metrics
+
+All of these features can be managed from the new Data Management tab, and these are just the first steps in our ongoing commitment to making organizing your data easier.
+
```
```suggestion
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+![PostHog - data management tab](../images/blog/data-management-feature/data_management_tab.png)
+
+## How to create event and event property definitions
```
```suggestion
## Exploring the new Data Management features
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+[Events](/docs/user-guides/events) are the basic building blocks of data that is sent to PostHog. Events can capture a wide variety of different interactions, from a new user signing up to capturing a server event when an API endpoint is called.
+
+[Event Properties](/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
+
```
```suggestion

At its core, our new Data Management experience is all about making it easier to define and categorise these elements, no matter what part of the tool you're currently using. Let's look at some examples.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+[Event Properties](/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
+
+
```
```suggestion
As part our new Data Management experience, we've introduced three different ways to define them.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+### Creating definitions in the new Data Management Tab
```
```suggestion
### 2. From the new Data Management tab
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+[WIP screenshot]
+
+### Creating definitions with the API
```
```suggestion
### 3. Via the API
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+### Creating definitions in the Insight Builder (on the fly!)
+
+Building an insight and see an event or event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
```
```suggestion
This is a great way to proactively improve descriptions during every day use. See an Event or Event Property that could be more descriptive? There's no excuse not to fix it then and there.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,96 @@
+
+### Creating definitions with the API
+
+You can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
```
```suggestion
Finally, you can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,92 @@
+
+3. Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page. 
+
+We highly recommend using tags to organize events coming in from different parts of your product. 
```
I think it would be useful if we briefly talked about how we use tags at PostHog and provide a few examples.

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,92 @@
+- Easily manage all your Events, Event Properties and Actions in the Data Management tab
+- Debug your incoming events using in the Live Events tab
+
+These are powerful tools for organizing your data, but we're just scratching the surface. [AV: Let's elaborate a little more here on the roadmap. Only needs to be top-level]
```
We need to a few little details about what's on our roadmap; get people excited about what's coming soon.

andyvan-ph replied:
```suggestion
These are powerful tools for organizing your data, but we're just scratching the surface. 
```

andyvan-ph commented:
@alexkim205 @paolodamico: FYI, I took the copy offline to give it a bit of rejig as I was finding it hard to understand exactly what we've changed, how and why. All the same basic components from the draft are here, they're. just in a different order and slightly reformatted.

A bit of a summary of changes:

- I've taken the bullet point list in the intro and moved it to the end as a recap
- I've explained some of the semantic changes we've made earlier into piece as it felt confusing mentioning them towards the end
- I've put more emphasis on how you can edit definitions in the Insight Builder, while also explaining the other ways you can do that.

I hope this looks good from your POV and let me know if I've introduced any misconceptions or inaccuracies.

alexkim205 commented:
@andyvan-ph This flows so much better, really appreciate the help. I made a few minor additions and some rewording but feel free to edit as you see fit.

Changes:
- added few examples in best practices section
- added future roadmap items
- add live events image
- remove sensitive total events image
- add blurb about 30 day volume and query volume metrics

Overall happy with how this turned out. LGTM!

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+![PostHog - data management tab](../images/blog/data-management-feature/data_management_tab.png)
+
+Data Management introduces more ways to organize and categorize your events. Event definitions can now be organized using tags and we've created the "Verified" marker, which tells collaborators that an event should be favored over other similar events. Verified events are also prioritized within filters and other selection components, ensuring people use the correct events when creating insights. ...
```
```suggestion
Data Management introduces more ways to organize and categorize your events. Event definitions can now be organized using tags and we've created the "Verified" marker, which tells collaborators that an event should be favored over other similar events. 

Verified events are also prioritized within filters and other selection components and we've introduced to two sortable "freshness" indicators (30-day volume and query volume), all of which make it easier to select the best Events when creating Insights. 
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+Data Management introduces more ways to organize and categorize your events. Event definitions can now be organized using tags and we've created the "Verified" marker, which tells collaborators that an event should be favored over other similar events. Verified events are also prioritized within filters and other selection components, ensuring people use the correct events when creating insights. ...
+
+Existing users should note we've made a few important changes to how data is organized, too. "Event Stats" and "Property Stats" have been renamed "Events" and "Event Properties," and we've moved them into the Data Management tab alongside "Actions".
+
+Alongside Data Management, we've moved "Events" into a new Live Events tab, which makes it easy to debug incoming event data by streaming it in near real time.
```
```suggestion
As part of this overhaul, we've made a few important changes to how data is organized, too. "Event Stats" and "Property Stats" have been renamed "Events" and "Event Properties," and we've moved them into the Data Management tab alongside "Actions". The "Events & Actions" tab has been renamed 'Live Events', which makes it easy to debug incoming event data by streaming it in near real time.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+Building an insight and see an Event or Event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
+
+Just hover over the Event Property in the dropdown...
```
```suggestion
Just hover over the Event property in the dropdown...
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+To help you in your journey to data management nirvana, here are a few rules of thumb to consider when using these new features.
+
+1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
```
```suggestion
1. The more definitions you add, the easier it will be for your teammates to understand what each Event and property represents.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+Not only are we giving you more ways to organize and define events, we're also making it easy to edit them wherever you are in the product.
+
+Building an insight and see an Event or Event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
```
```suggestion
Building an insight and see an event or event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+Alternatively, click on 'View' in the definition hover card and you'll be redirected to the new Data Management page, where you can explore all the Events and Event Properties you've created.
```
```suggestion
Alternatively, click on 'View' in the definition hover card and you'll be redirected to the new Data Management page, where you can explore all the Events and Event properties you've created.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PostHog instance sending sign up events as `sign up`. Or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, pote ...
```
```suggestion
Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self-explanatory, right? Not necessarily. Imagine you have a second app pushing data to the same PostHog instance sending sign up events as `sign up`; or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, potentially leading to incorrect conclusions and frustration. Describing these events properly can ensure your teammates find exactly what they're looking for.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+- Organize your definitions with tags
+- Manage event definitions on the fly while building insights
+- Verify trusted definitions which are prioritized in the UI
+- Easily manage all your Events, Event Properties and Actions in the Data Management tab
```
```suggestion
- Easily manage all your Events, Event properties and Actions in the Data Management tab
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+
+- Role-based ownership of definitions
+- Personally identifiable information (PII) rules
+- Anomaly detection for malformed Events and Properties
```
```suggestion
- Anomaly detection for malformed events and properties
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+- Anomaly detection for malformed Events and Properties
+- Person and Group Property definitions
+
+As always, we welcome any feedback you may have about this feature. Feel free to open an issue in [our Github repo](https://github.com/PostHog/posthog) or give us a shout in our [PostHog community Slack channel](/slack). Or join us directly for a [call](https://calendly.com/posthog-feedback) with our Product & Engineering team.
```
```suggestion
As always, we welcome any feedback you may have about this feature. Feel free to open an issue in [our Github repo](https://github.com/PostHog/posthog), give us a shout in our [PostHog community Slack channel](/slack),  or join us directly for a [call](https://calendly.com/posthog-feedback) with our Product & Engineering team.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+categories: ["Product updates", "Launch week"]
+---
+
+PostHog is growing fast. In just the last year (since we started measuring total events), we measured ~36.5B total events ingested in PostHog Cloud alone, and hundreds of self-hosted users reached the 1M total ingested events milestone!
```
```suggestion
PostHog is growing fast. In just the last year we've measured ~36.5B total events ingested in PostHog Cloud alone, and hundreds of self-hosted users reached the 1M total ingested events milestone.
```

paolodamico commented on netlify.toml:
```diff
@@ -1261,3 +1261,45 @@
+
+
+# Added: 2022-03-15
+[[redirects]]
```
Hmm these changes seem unrelated.

charlescook-ph commented:
This looks great, nice job @alexkim205 !

andyvan-ph commented:
@lottiecoxon Not sure if you'd looked at this one yet, but basically this is all about categorizing and organizing information, which gave me a few different ideas:

1. Something with filing cabinets. Not especially imaginative but, I imagine some kind of office scene with hedgehogs filing papers.
 
2. A big more fun, we could have a hedgehog sitting behind a desk checking in a long queue, all of whom are wearing hats to signify event tags, e.g. 'user sign up', 'insight viewed', 'clicked button', 'Verified event' and so on.

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,97 @@
+2. Data decays quickly, and most of the time fails to stay up-to-date with instrumentation.
+3. Event-level context in insights and dashboards is difficult to share with external people and new team members.
+
+We set out to solve these pain points and the result is the new Data Management experience we're introducing today, starting with the new Data Management tab.
```
Awaiting confirmation on the date

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,101 @@
+
+Data Management introduces more ways to organize and categorize your events. Event definitions can now be organized using tags and we've created the 'Verified' marker, which tells collaborators that an event should be favored over other similar events. 
+
+Verified events are also prioritized within filters and other selection components, and we've introduced to two sortable "freshness" indicators (30-day volume and query volume) to make it easier to choose the correct events when creating Insights.
```
```suggestion
Verified events are prioritized within filters and other selection components, and we've introduced to two sortable "freshness" indicators (30-day volume and query volume) to make it easier to choose the correct events when creating Insights.
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,101 @@
+
+To help you in your journey to data management nirvana, here are a few rules of thumb to consider when using these new features.
+
+1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
```
```suggestion
**1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.**
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,101 @@
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self-explanatory, right? Not necessarily. Imagine you have a second app pushing data to the same PostHog instance sending sign up events as `sign up`; or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate betw ...
+
+2. Official organization-wide events should be verified so that teammates can discover and use them easily. 
```
```suggestion
2. **Official organization-wide events should be verified so that teammates can discover and use them easily.** 
```

andyvan-ph commented on contents/blog/data-management-feature.md:
```diff
@@ -0,0 +1,101 @@
+
+Verified events will be prioritized in filters and other selection components to signal to the rest of the team that this event should be used in favor of other similar events. An example where this comes in handy is if there are two events that sound familiar, `signed up` and `signed_up`. Verifying the first tells others very explicitly that one is still being used and the other has gone stale.
+
+3. Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page. 
```
```suggestion
**3. Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page.** 
```
